app/models/customers_item.rb

```diff
@@ -0,0 +1,8 @@
+class CustomersItem < ApplicationRecord
+  belongs_to :item
+  belongs_to :customer
+
+  def self.count_purchases(item)
+    where('item_id = ?', item.id).count
```

Comment:
This could also live in the Item model and you can use associations: `customers.distinct.count`